modules/extensions-spring-support/src/test/java/org/mule/runtime/module/extension/internal/resources/ExtensionModelJsonGeneratorTestCase.java

```diff
@@ -54,31 +56,56 @@ public class ExtensionModelJsonGeneratorTestCase extends FileGenerationParameter
   private static final boolean UPDATE_EXPECTED_FILES_ON_ERROR =
       getBoolean(SYSTEM_PROPERTY_PREFIX + "extensionModelJson.updateExpectedFilesOnError");
 
+  private static final String TEST_GROUP_ID = "org.mule.tests";
+  private static final String VERSION = "4.5.0";
```

Comment:
externalize to get the actual version or put something that has nothing to do with the runtime versioning